ansible/roles/django_db/tasks/main.yml

```diff
@@ -22,6 +22,8 @@
     - reset-admin-password --password admin
   become: true
   become_user: '{{ pulp_user }}'
+  tags:
+    - skip_ansible_lint
```

Comment:
what does this do to the PR?

Reply:
this part let travis pass the ansible-list part,
otherwise getting
~~~
ANSIBLE0012 Commands should not change things if nothing needs doing
/home/user/devel/devel/ansible/roles/django_db/tasks/main.yml:14
Task/Handler: Run all migrations
~~~
In my opinion we want to run migration to ensure everything prepared.

Reply:
@bmbouter I think we should at least merge the tag commit; atm CI is blocked by it: https://travis-ci.com/pulp/devel/jobs/146982641 https://github.com/pulp/devel/pull/183/commits/fe2d2a712273726e358aaccc95c8fd6a86e5060c